### DISCUSSION TITLE
Validate the ResizedImage

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5139] [Help]
Im using magickwand to resize the image 

Is there any way to validate the Resized Image ? 
I need to check the both original and resized image is same or not in program

**Comment by snibgo** [comment]
I don't understand. What would make an image "invalid"?

You can check that the file is valid with "magick yourfile.ext -verbose info:" and check for errors.

You can repeat the resize, in the opposite direction, and use "magick compare" to evaluate whether the images are similar.
